Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "petgraph"
-version = "0.6.5"
+version = "0.7.0"
```

Comment:
0.7.0 will be created from the `next` branch: https://github.com/petgraph/petgraph/pull/567
Use 0.6.6

Reply:
Gotcha; I've swapped it to 0.6.6